### PR TITLE
Minor component generator updates

### PR DESCRIPTION
**Pull request by lyzhan7**
### Platforms Impacted
- [x] iOS
- [x] macOS
- [x] win32 (Office)
- [x] windows
- [x] android

### Description of changes

Updated the readme with some updated instructions + other known issues based on comments in PRs #1761 and #1779 and made some minor updates. 

### Verification

Added a commit (that I later reverted so it doesn't get checked in) with an updated example of what gets generated.

### CODE REVIEW

File: apps/component-generator/README.md [hunk 1]
```diff
@@ -4,14 +4,14 @@ It's used to automate process of component creation.
 
 ## Setup
 
-1. Run in your command prompt `npm i -g gulp-cli`
-2. Run `npm i` inside the component-generator directory
+1. Run in your command prompt `npm i -g gulp-cli`. May need to run this with `sudo`
```
Comment (harrieshin) [inline apps/component-generator/README.md:7]: is "sudo" only a mac thing? lol i don't know how windows does admin permission.  run as admin terminal window?

Comment (Saadnajmi) [reply apps/component-generator/README.md:7]: Yep. You have to open command prompt as an administrator, which you can do from it's context menu.

Comment (rohanprasadofficial) [reply apps/component-generator/README.md:7]: For me , it ran without admin permissions on windows.

Comment (lyzhan7) [reply apps/component-generator/README.md:7]: I'll add a comment specifying that the `sudo` part is only for mac!

**Review by Saadnajmi — APPROVED** [review]

**Review by ValentinaKozlova — APPROVED** [review]

**Comment by ValentinaKozlova** [comment]
Thank you for updating documentation

**Pull request merged**